### PR TITLE
Add CK CI starter template

### PR DESCRIPTION
**Pull request by stonepreston**

### CODE REVIEW

File: workflow-templates/ck-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@main
+        with:
+          provider: vsphere
+          credentials-yaml: ${{ secrets.CREDENTIALS_YAML }}
+          clouds-yaml: ${{ secrets.CLOUDS_YAML }}
+          bootstrap-constraints: "arch=amd64 cores=2 mem=4G"
+          bootstrap-options: "${{ secrets.FOCAL_BOOTSTRAP_OPTIONS }} --model-default datastore=vsanDatastore --model-default primary-network=VLAN_2763"
+      - name: Run test
+        run: tox -e integration
```
Comment (kwmonroe) [inline workflow-templates/ck-ci.yaml]:
```suggestion
```

Split this into a different workflow file since this won't always work (eg k8s charms)

**Review by kwmonroe — APPROVED** [review]
LGTM if we split the integration test out.

File: workflow-templates/integration.yaml [hunk 2]
```diff
@@ -1,17 +1,7 @@
...
-    name: Inclusive Naming
-    uses: canonical-web-and-design/Inclusive-naming/.github/workflows/woke.yaml@main
-    with:
-      fail-on-error: "true"
-
-  lint-unit:
-    name: Lint Unit
-    uses: charmed-kubernetes/workflows/.github/workflows/lint-unit.yaml@main
-
   integration-test:
```
Comment (addyess) [inline workflow-templates/integration.yaml]: Should we call it 

```suggestion
  vsphere-integration-test:
```

That way maybe we add different types of integration-tests in here

**Review by addyess — APPROVED** [review]
LGTM

**Pull request merged**